RecoJets/JetProducers/plugins/PUFilter.h

```diff
@@ -0,0 +1,54 @@
+/* -*- C++ -*-
```

Comment:
move contents of this file to  RecoJets/JetProducers/plugins/PUFilter.cc 
(and remove this file)